lib/portlayer/exec/exec.go

```diff
@@ -41,7 +45,7 @@ var (
 	}
 )
 
-func Init(ctx context.Context, sess *session.Session, source extraconfig.DataSource, _ extraconfig.DataSink) error {
+func Init(ctx context.Context, sess *session.Session, source extraconfig.DataSource, _ extraconfig.DataSink, self types.ManagedObjectReference) error {
```

Comment:
self reference in this init function is kinda confusing. What does it mean for anybody who sees this function signature for the first time?

Reply:
Yup - general requirement for function doc.